codespell.txt

```diff
@@ -0,0 +1,4 @@
+hel
```

Comment:
Sorry I failed to understand this. Is this a whitelist for misspell?

Reply:
`codespell` is a new Python based spell checker that runs on `pre-commit`.  This is the `codespell` dictionary and it found some new spelling mistakes.  I added docs on this